modules/survey_accounts/php/NDB_Form_survey_accounts.class.inc

```diff
@@ -205,8 +205,8 @@ class NDB_Form_survey_accounts extends NDB_Form
 
             $msg_data = array(
                 'study'     => $config->getSetting("title"),
-                'url'       => $www['url'] . '/survey.php?key=' . $key,
-                'EmailForm' => $values['EmailContent']
+                'url'       => $www['url'] . '/survey.php?key=' . urlencode($key),
+                'EmailForm' => $values['email_dialog']
```

Comment:
This change was already merged in 16.04 in https://github.com/aces/Loris/pull/1829